tests/rest/client/v2_alpha/test_account.py

```diff
@@ -250,10 +250,23 @@ def _validate_token(self, link):
         # Remove the host
         path = link.replace("https://example.com", "")
 
+        # Load the password reset confirmation page
         request, channel = self.make_request("GET", path, shorthand=False)
         self.render(request)
         self.assertEquals(200, channel.code, channel.result)
 
+        # Replace the path with the confirmation path
+        path = re.sub(
+            "^/_matrix.*submit_token",
+            "/_matrix/client/unstable/password_reset/email/submit_token_confirm",
+            path,
+        )
+
+        # Confirm the password reset
+        request, channel = self.make_request("POST", path, shorthand=False)
+        self.render(request)
+        self.assertEquals(200, channel.code, channel.result)
```

Comment:
surely this isn't sending any of the fields in the POST request, so should fail with a 400?

Reply:
Ah, well this happens to succeed as it seems request arguments and form data in the request body both end up in `request.args` in twisted.

But your right, this test isn't sending a request the same way a client would send it.

Reply:
Fixed in b12c3d037 :tada: